src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -63,11 +63,12 @@ export const RegistrationFormActions = ({
         ['registration', updateRegistrationResponse.data.identifier],
         updateRegistrationResponse.data
       );
+      const newErrors = validateForm(updateRegistrationResponse.data);
       dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
+      resetForm({ values: updateRegistrationResponse.data, errors: newErrors });
```

Comment:
Ah, må bruke `resetForm` for å nullstille `dirty`? Det gir jo mening💡 
Om jeg har forstått det riktig vil jeg tro det går greit om de to nye linjene her legges i `else` på linje 70? Dette for å unngå at det gjøres hver gang 🤔